css/mystyle.css

```diff
@@ -223,7 +249,16 @@ color: #838994;
     .social{
     	padding: 3em 0;
     }
+    .assign .illustration li img {
```

Comment:
This makes it so the project pictures will resize and get smaller when the screen gets smaller.